[2025-02-09T22:32:49.559+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-09T22:32:49.571+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mgp-airflow1.docker_command_hello manual__2025-02-09T22:32:45.575928+00:00 [queued]>
[2025-02-09T22:32:49.575+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mgp-airflow1.docker_command_hello manual__2025-02-09T22:32:45.575928+00:00 [queued]>
[2025-02-09T22:32:49.576+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-09T22:32:49.583+0000] {taskinstance.py:2889} INFO - Executing <Task(DockerOperator): docker_command_hello> on 2025-02-09 22:32:45.575928+00:00
[2025-02-09T22:32:49.591+0000] {standard_task_runner.py:72} INFO - Started process 278 to run task
[2025-02-09T22:32:49.596+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'mgp-***1', 'docker_command_hello', 'manual__2025-02-09T22:32:45.575928+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/mgpMap.py', '--cfg-path', '/tmp/tmp0_u6li6i']
[2025-02-09T22:32:49.598+0000] {standard_task_runner.py:105} INFO - Job 25: Subtask docker_command_hello
[2025-02-09T22:32:49.614+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-09T22:32:49.691+0000] {task_command.py:467} INFO - Running <TaskInstance: mgp-airflow1.docker_command_hello manual__2025-02-09T22:32:45.575928+00:00 [running]> on host 8c9cea95fee4
[2025-02-09T22:32:49.795+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mgp-***1' AIRFLOW_CTX_TASK_ID='docker_command_hello' AIRFLOW_CTX_EXECUTION_DATE='2025-02-09T22:32:45.575928+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-09T22:32:45.575928+00:00'
[2025-02-09T22:32:49.797+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-09T22:32:49.948+0000] {docker.py:495} INFO - ::group::Pulling docker image python:3.8-slim
[2025-02-09T22:32:51.829+0000] {docker.py:509} INFO - 3.8-slim: Pulling from library/python
[2025-02-09T22:32:52.208+0000] {docker.py:509} INFO - 14c9d9d19932: Pulling fs layer
[2025-02-09T22:32:52.210+0000] {docker.py:509} INFO - d847ad1879f2: Pulling fs layer
[2025-02-09T22:32:52.212+0000] {docker.py:509} INFO - 56d45b8e1404: Pulling fs layer
[2025-02-09T22:32:52.214+0000] {docker.py:509} INFO - 0cd45dda32d5: Pulling fs layer
[2025-02-09T22:32:52.214+0000] {docker.py:509} INFO - 0cd45dda32d5: Waiting
[2025-02-09T22:32:52.782+0000] {docker.py:509} INFO - 14c9d9d19932: Downloading
[2025-02-09T22:32:52.868+0000] {docker.py:509} INFO - d847ad1879f2: Downloading
[2025-02-09T22:32:52.903+0000] {docker.py:509} INFO - 56d45b8e1404: Downloading
[2025-02-09T22:32:53.033+0000] {docker.py:509} INFO - d847ad1879f2: Download complete
[2025-02-09T22:32:53.382+0000] {docker.py:509} INFO - 56d45b8e1404: Verifying Checksum
[2025-02-09T22:32:53.383+0000] {docker.py:509} INFO - 56d45b8e1404: Download complete
[2025-02-09T22:32:53.551+0000] {docker.py:509} INFO - 14c9d9d19932: Verifying Checksum
[2025-02-09T22:32:53.552+0000] {docker.py:509} INFO - 14c9d9d19932: Download complete
[2025-02-09T22:32:53.607+0000] {docker.py:509} INFO - 14c9d9d19932: Extracting
[2025-02-09T22:32:53.661+0000] {docker.py:509} INFO - 0cd45dda32d5: Verifying Checksum
[2025-02-09T22:32:53.664+0000] {docker.py:509} INFO - 0cd45dda32d5: Download complete
[2025-02-09T22:32:54.830+0000] {docker.py:509} INFO - 14c9d9d19932: Pull complete
[2025-02-09T22:32:54.832+0000] {docker.py:509} INFO - d847ad1879f2: Extracting
[2025-02-09T22:32:54.927+0000] {docker.py:509} INFO - d847ad1879f2: Pull complete
[2025-02-09T22:32:54.929+0000] {docker.py:509} INFO - 56d45b8e1404: Extracting
[2025-02-09T22:32:55.422+0000] {docker.py:509} INFO - 56d45b8e1404: Pull complete
[2025-02-09T22:32:55.424+0000] {docker.py:509} INFO - 0cd45dda32d5: Extracting
[2025-02-09T22:32:55.436+0000] {docker.py:509} INFO - 0cd45dda32d5: Pull complete
[2025-02-09T22:32:55.448+0000] {docker.py:504} INFO - Digest: sha256:1d52838af602b4b5a831beb13a0e4d073280665ea7be7f69ce2382f29c5a613f
[2025-02-09T22:32:55.449+0000] {docker.py:504} INFO - Status: Downloaded newer image for python:3.8-slim
[2025-02-09T22:32:55.454+0000] {docker.py:511} INFO - ::endgroup::
[2025-02-09T22:32:55.455+0000] {docker.py:367} INFO - Starting docker container from image python:3.8-slim
[2025-02-09T22:32:55.504+0000] {docker.py:375} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2025-02-09T22:32:56.619+0000] {docker.py:438} INFO - hello
[2025-02-09T22:32:56.823+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-09T22:32:56.823+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=mgp-***1, task_id=docker_command_hello, run_id=manual__2025-02-09T22:32:45.575928+00:00, execution_date=20250209T223245, start_date=20250209T223249, end_date=20250209T223256
[2025-02-09T22:32:56.875+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-09T22:32:56.903+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-09T22:32:56.909+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
